src/lib/Scenes/Show/Screens/MoreInfo.tsx

```diff
@@ -16,6 +19,7 @@ const ListHeaderText = styled(Serif)`
 
 interface Props extends ViewProperties {
   show: MoreInfo_show
+  showID: string
```

Comment:
I don't think this `showID` isn't being used since you have it defined in `ShowMoreInfoRenderer`